package/Gemfile.lock

```diff
@@ -79,7 +79,7 @@ GEM
       ffi-compiler (>= 1.0, < 2.0)
     http_parser.rb (0.6.0)
     httpclient (2.8.3)
-    json (2.3.0)
+    json (2.3.1)
```

Comment:
Gemfile.lock is much like go.sum, its not something we should edit directly. We should be specifying the version here: https://github.com/rancher/fluentd/blob/master/package/Gemfile#L5
Unfortunate that none of the packages actually have versions specified there, because as soon as you do a `bundle install` it's going to try to bump all of them. I think best approach here is to look at this lock file and specify all the versions in the Gemfile, and then just bump json gem.

You can try just doing something like `bundle update json` and see if it bumps just that one, but it will likely update to a much newer version and then we get into dependency hell. `bundle install` should resolve that sort of thing for you.

Reply:
Bundler has some good docs: https://bundler.io/v2.1/guides/using_bundler_in_applications.html